src/frontend/mame/ui/filesel.cpp

```diff
@@ -491,8 +491,8 @@ menu_file_selector::file_selector_entry *menu_file_selector::append_dirent_entry
 
 void menu_file_selector::append_entry_menu_item(const file_selector_entry *entry)
 {
-	const char *text = nullptr;
-	const char *subtext = nullptr;
+	std::string text;
+	std::string subtext;
```

Comment:
If you're using temporary `std::string` that you want used destructively, you must use `std::move` when you make the call.  If you don't want the explicit `std::move`, you could have left these as `const char *` and just initialised to `""` instead of `nullptr` here.